Add UK comparison plot

kathsherratt opened the pull request:
This PR adds markdown for a UK comparison plot (attached) based on the daily Rt update from deaths/admissions/cases, plus some explanatory text. 

I don't know where this will best fit in to the website structure / UK page though. Because of that for now I have kept this in a separate Rmd - but expect we will want to slot this into the UK post and delete these separate files.

Sharing this as a first draft, please let me know how best to adapt this.

![image](https://user-images.githubusercontent.com/62290797/97817883-038ace80-1c97-11eb-9944-66ae3e5fa979.png)

seabbs reviewed (commented):
This is all looking good. I'd suggest downloading the data from the dataverse as that is more stable long term but otherwise :+1:. 

Can talk about how to integrate this in a call or happy to see a first draft (it should be essentially a case of copying this stuff in to the current Rmd. 

It might be nice to see a table of latest Rt estimates by data source and region (using `DT` for example (see EpiNow2 templates for an example).

kathsherratt commented:
Sorry this has taken a madly long time to get to doing. Looks like that has created merge conflicts... this is my bad, I'm sorry. I am doing a botch job of resolving so any help appreciated!

After review:
- I have added the above work to the main UK page
- I have added a new table with side by side Rt estimates as suggested.
- I have split out the plots. The national plot looks better. The NHS regions plot still has a bit of an odd width/height ratio (too squat!) but I haven't quite been able to figure out how to display this nicely. Please suggest if you have a better way!
- I haven't replaced the link with the dataverse as it looks like we don't publish the summary csv on the dataverse (only an [rt.csv](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/4L3OKY) which doesn't seem to attribute data source. Instead I am using what is on github [here](https://github.com/epiforecasts/covid-rt-estimates/blob/master/subnational/united-kingdom/collated/rt/summary_latest.csv)).

Also tagging for review by @sbfnk .

seabbs commented on _posts/national/united-kingdom/united-kingdom.Rmd:
```diff
@@ -36,8 +36,7 @@ source(here::here("utils/subnational_defaults.R"), local = TRUE)
 
-<br>
-`r widget_caption`
+<br> `r widget_caption`
```
There is an authorship section in the YAML. I suggest updating to that used in the paper. There is also a citation section where the paper can be linked. Check out the docs for details (https://rstudio.github.io/distill/citations.html)

seabbs replied:
I'd also suggest mentioned where the methodology for this is outlined (i.e your paper) at the top somewhere.

seabbs commented on _posts/national/united-kingdom/united-kingdom.Rmd:
```diff
@@ -62,3 +61,121 @@ summary_path <- file.path("covid-rt-estimates/subnational", folder, "cases/summa
+
+We can use this feature of each data source to explore how transmission is spreading in different parts of the population. For example, in the UK in August, most new infections were spread between young people, typically less vulnerable to severe disease. This meant the Rt from all test-positive cases rose quicker over that period than the Rt calculated from patients in hospital.
+
+By tracking these comparisons, we can understand how Covid-19 is affecting the most vulnerable populations. To see this more clearly, we plot and compare Rt estimates from each data source.
```
maybe reduce into two paragraphs as last two seem to repeat information? If possible use a bit citation for the paper link (put in library.bib at root and in the UK folder) and use a cite key as for other references.

seabbs commented on _posts/national/united-kingdom/united-kingdom.Rmd:
````diff
@@ -62,3 +61,121 @@ summary_path <- file.path("covid-rt-estimates/subnational", folder, "cases/summa
+To explore in more depth how Rt from different data sources can be used to understand transmission dynamics across the population, see our pre-print (currently in peer-review) [here](https://doi.org/10.1101/2020.10.18.20214585).
+
+```{r, echo=FALSE}
+library(magrittr)
````
check all deps here are listed in the description and ideally load at the top clearly.

seabbs commented on _posts/national/united-kingdom/united-kingdom.Rmd:
````diff
@@ -62,3 +61,121 @@ summary_path <- file.path("covid-rt-estimates/subnational", folder, "cases/summa
+
+To explore in more depth how Rt from different data sources can be used to understand transmission dynamics across the population, see our pre-print (currently in peer-review) [here](https://doi.org/10.1101/2020.10.18.20214585).
+
+```{r, echo=FALSE}
````
can alter plot size with fig.width, fig.heigth etc if required.

seabbs commented on _posts/national/united-kingdom/united-kingdom.Rmd:
````diff
@@ -62,3 +61,121 @@ summary_path <- file.path("covid-rt-estimates/subnational", folder, "cases/summa
 ```{r,  child = system.file("templates/_regional-summary.Rmd", package = "EpiNow2")}
 ```
+
+## Comparison of cases, admissions, and deaths data
````
is this in the correct order? I would have thought this would replace the templates or at least come before?

seabbs reviewed (approved):
Looks good to me. Suggest thinking about the flow a bit as I am not sure we should have this at the bottom as it seems to contain most of the info people need/perhaps should go below the national level summary? Alternatively could completely replace.

seabbs commented:
Can you screenshot the rendered site - no local checkout right now.

kathsherratt commented:
Here is a screenshot (in two halves) as is. I'll work through your comments - thanks.

![image](https://user-images.githubusercontent.com/62290797/100224498-cff93800-2f14-11eb-88a6-e494a076b783.png)

![image](https://user-images.githubusercontent.com/62290797/100224530-dab3cd00-2f14-11eb-9000-2ced98f4ce89.png)

kathsherratt commented:
Edited following review:
+ Restructured the section order as below, but still happy to move around:
   1. interactive summary
   2.  national summary template
   3.  comparison
   4. regional summary template
+ Shortened text around comparison
+ Figs separated to get height/width
+ Added authorship / citation in YAML at the top - I think this is was @seabbs intended, but I may be getting the format 
wrong
+ Added paper ref as in text bib citation (already in the .bib files)

However I am having some problem when trying to knit the Rmd. Looks like it gets stuck processing the regional summary template: see error message attached. Between this and the last commit I have not actually changed any inputs to the regional summary template, but I must be missing something. 

Uploading this commit anyway so you can see and suggest any further changes while I try and work out what is going wrong with the knitting bit.
![image](https://user-images.githubusercontent.com/62290797/100380584-0a470000-300f-11eb-99a2-610efd8bcc4a.png)

seabbs commented:
sorry I mad to scrub the repo history and it has made this PR very confused. Will copy the changes across into a new PR and delete this branch.
